ajax/libs/micromustache/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "micromustache",
+  "filename": "micromustache.min.js",
+  "version": "3.0.4",
+  "description": "A faster and smaller subimplementation of the {{mustache}} template engine for JavaScript",
+  "keywords": [
+    "template",
+    "mustache",
+    "string processing",
+    "string interpolation",
+    "template engine",
+    "microlibrary"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/userpixel/micromustache.git"
+  },
+  "author": "Alex Ewerlöf",
+  "license": "MIT",
+  "homepage": "",
```

Comment:
We usually add official website here not git repo.
Is there a website for this library?
If it isn't, I will remove the `"homepage"`.
Thanks!